x-pack/plugin/core/src/main/resources/synthetics-settings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "template": {
+    "settings": {
+      "index": {
+        "lifecycle": {
+          "name": "synthetics"
+        },
+        "codec": "best_compression",
+        "query": {
+          "default_field": ["message"]
```

Comment:
This one feels wrong since synthetics don't have a message?

Reply:
Good call, I'll remove the `default_field` and stick with implicit "all" then, since it doesn't seem like there's a good alternative for this?